build/Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM centos:7
 MAINTAINER Ansible Service Broker Community
 
-RUN useradd -M -s /usr/sbin/nologin -u 1001 -g 0 ansibleapp
+RUN mkdir /opt/ansibleapp
+RUN useradd -M -b /opt/ansibleapp -s /usr/sbin/nologin -u 1001 -g 0 ansibleapp
+RUN chown -R 1001:0 /opt/ansibleapp
+
+RUN yum -y update \
+ && yum -y install epel-release centos-release-openshift-origin \
+ && yum -y install origin origin-clients net-tools bind-utils ansible \
```

Comment:
I was surprised to see these as additions since they're core dependencies, so I was wondering how this container could have ever worked without them. @jmrodri explained `make deploy` was actually pulling the old `ansibleplaybookbundle/ansible-service-broker-asb` that had been produced by the ansible-container builds instead of new builds produced by this Dockerfile before this change. Makes sense why they needed to be added in this change now.